actionmailbox/app/controllers/action_mailbox/ingresses/amazon/confirmations_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module ActionMailbox
+  module Ingresses
+    module Amazon
+      class ConfirmationsController < BaseController
```

Comment:
Consider modeling as a Subscription resource. The SNS SubscriptionConfirmation message maps to `#create` and creates a subscription. Then the SNS UnsubscribeConfirmation message would naturally map to `#destroy`.